apps/web/src/app/[locale]/globals.css

```diff
@@ -58,36 +64,39 @@
     background-attachment: local, local, scroll, scroll;
   }
 
-  @media (prefers-color-scheme: dark) {
-    .scroll-shadows {
-      background:
+  .dark .scroll-shadows {
+    background:
       /* Shadow Cover TOP */
+      linear-gradient(theme("colors.dark-bg") 30%, theme("colors.dark-bg / 0%"))
+        center top,
+      /* Shadow Cover BOTTOM */
         linear-gradient(
-            theme("colors.stone.900") 30%,
-            theme("colors.stone.900 / 0%")
-          )
-          center top,
-        /* Shadow Cover BOTTOM */
-          linear-gradient(
-            theme("colors.stone.900 / 0%"),
-            theme("colors.stone.900") 70%
-          )
-          center bottom,
-        /* Shadow TOP */
-          radial-gradient(
-            farthest-side at 50% 0,
-            theme("colors.stone.100 / 30%"),
-            theme("colors.stone.100 / 0%")
-          )
-          center top,
-        /* Shadow BOTTOM */
-          radial-gradient(
-            farthest-side at 50% 100%,
-            theme("colors.stone.100 / 30%"),
-            theme("colors.stone.100 / 0%")
-          )
-          center bottom;
-    }
+          theme("colors.dark-bg / 0%"),
+          theme("colors.dark-bg") 70%
+        )
+        center bottom,
+      /* Shadow TOP */
+        radial-gradient(
+          farthest-side at 50% 0,
+          theme("colors.stone.400 / 30%"),
```

Comment:
I think we should use the brand / gray palette that the light mode site also uses, or create a new custom palette for dark mode. 

The stone color palette looks nice, but it's quite different from the light theme colors